code/modules/events/holiday/halloween.dm

```diff
@@ -53,6 +53,46 @@
 	if(IsMultiple(activeFor, 4))
 		spawn_meteors(3, meteorsSPOOKY) //meteor list types defined in gamemode/meteor/meteors.dm
 
+//Creepy clown invasion
+/datum/round_event_control/creepy_clowns
+	name = "Clowns"
+	typepath = /datum/round_event/creepy_clowns
+	holidayID = HALLOWEEN
+	weight = 20
+	earliest_start = 0
+
+/datum/round_event/creepy_clowns
+	endWhen = 40
+
+/datum/round_event/creepy_clowns/start()
+	for(var/mob/living/carbon/human/H in living_mob_list)
+		if(!H.client || !istype(H))
+			return
+		H << "<span class='danger'>Honk...</span>"
+		H << 'sound/spookoween/scary_clown_appear.ogg'
+		var/turf/T = get_turf(H)
+		if(T)
+			new /obj/effect/hallucination/delusion(H.loc,H,force_kind="clown",duration=300,skip_nearby=0)
+
+/datum/round_event/creepy_clowns/tick()
+	if(IsMultiple(activeFor, 3))
+		for(var/mob/living/carbon/human/H in living_mob_list)

```

Comment:
It looks like you can just blindly cast to mob/living really, or even mob, you're not using any specific human vars